README.md

```diff
@@ -33,8 +33,23 @@ ConstructorIo.init(this, config)
 // Set the user ID (for a logged in user) used for cross device personalization
 ConstructorIo.userId = "uid"
 ```
+## 4. Retrieving the client_id and session_id
+If you are retrieving results from your backend servers instead of direclty using our sdk, there are two parameters that you will want to forward along with your request.
 
-## 4. Request Autocomplete Results
+1. **Client Id**
+This Can be retrieved by calling **getClientId** function on the client sdk instance.
+```kotlin
+var constructorClientId = ConstructorIo.getClientId()
+```
+2. **Session Id**
+This Can be retrieved by calling **getSessionId** function on the client sdk instance.
+```kotlin
+var constructorSessionId = ConstructorIo.getSessionId()
+```
+
+In most cases, you will want to attach these parameters as the cookies **ConstructorioID_client_id** and **ConstructorioID_session_id** to requests to your backend servers.
```

Comment:
Are you trying to say that they'll need to add the two values as cookies so they can pass them later to their backend servers? If so can we rephrase it so it's clearer? 

Something like you will need to set these two values as cookies so you can pass them to your backend servers with the requests you're making. Not exactly that but just elaborate more 😅